presto-docs/src/main/sphinx/release/release-0.243.2.rst

```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.243.1
+===============
+
+General Changes
+------------
+General Changes
```

Comment:
Copy-paste error? Remove the duplicated title. Also, the number hyphens need to match the number of the characters on the last line.